2-Authorization-I/1-call-graph/SPA/src/app/app.module.ts

```diff
@@ -39,22 +66,35 @@ export function MSALInstanceFactory(): IPublicClientApplication {
 export function MSALInterceptorConfigFactory(): MsalInterceptorConfiguration {
   const protectedResourceMap = new Map<string, Array<string>>();
 
-  protectedResourceMap.set(protectedResources.graphMe.endpoint, protectedResources.graphMe.scopes);
-  protectedResourceMap.set(protectedResources.armTenants.endpoint, protectedResources.armTenants.scopes);
+  protectedResourceMap.set(
+    protectedResources.graphMe.endpoint,
+    protectedResources.graphMe.scopes
+  );
 
+   protectedResourceMap.set(
+     protectedResources.graphContacts.endpoint,
+     protectedResources.graphContacts.scopes
+   );
+  
   return {
     interactionType: InteractionType.Redirect,
     protectedResourceMap,
     authRequest: (msalService, httpReq, originalAuthRequest) => {
       const resource = new URL(httpReq.url).hostname;
-
-      let claim = msalService.instance.getActiveAccount()! &&
-          getClaimsFromStorage(
-            `cc.${msalConfig.auth.clientId}.${msalService.instance.getActiveAccount()?.idTokenClaims?.oid}.${resource}`
-          ) ? window.atob(
-            getClaimsFromStorage(
-              `cc.${msalConfig.auth.clientId}.${msalService.instance.getActiveAccount()?.idTokenClaims?.oid}.${resource}`
-            ))
+      let claim =
+        msalService.instance.getActiveAccount()! &&
+        getClaimsFromStorage(
```

Comment:
We will need to add some comments here (can be reused from the React sample), essentially to tell what the claims here is doing, what the claim challenge schema is and etc.